dimod/views.py

```diff
@@ -72,3 +74,239 @@ class IndexValuesView(abc.ValuesView):
     def __iter__(self):
         # Inherited __init__ puts the Mapping into self._mapping
         return iter(self._mapping._data.flat)
+
+class Bias(object):
```

Comment:
Wrapping all biases with a class seems awfully slow (especially in the average case where the bias is a primitive type like `int` or `float`).